fractions/microprofile/microprofile-jwt/pom.xml

```diff
@@ -36,6 +37,18 @@
     </resources>
   </build>
 
+  <dependencyManagement>
```

Comment:
Why is Arquillian dependency needed in the fraction pom ?

Reply:
It was used during some initial testing, but has been removed now.